net/vnstat/Makefile

```diff
@@ -26,12 +28,13 @@ include $(INCLUDE_DIR)/package.mk
 define Package/vnstat/Default
   SECTION:=net
   CATEGORY:=Network
-  URL:=http://humdi.net/vnstat/
+  URL:=https://humdi.net/vnstat/
 endef
 
 define Package/vnstat
 $(call Package/vnstat/Default)
   TITLE:=Console-based network traffic monitor
+  DEPENDS:=+libsqlite3
```

Comment:
Please don't add major new dependencies and use a **misleading** commit message like "Reorganize Makefile..." that makes it look like just housekeeping.